website/blog/2023-03-21-document-databases-definition-features-use-cases.md

```diff
@@ -0,0 +1,141 @@
+---
+slug: document-databases-definition-features-use-cases
+title: "Document Database: Definition, Features, Use Cases"
+author: Alexander Fashakin
+image: /img/blog/mongodb-alternatives.png
+description: >
+    Learn about NoSQL document databases, their features, benefits, use cases, and popular examples such as FerretDB, MongoDB, and Couchbase.
+unlisted: true
+---
+
+![FerretDB v0.9.1 - Minor Release](/img/blog/ferretdb-v0.9.1.jpg)
+
+Since their inception in the 1960s, databases have undergone significant changes, with relational databases like Oracle, MySQL, and PostgreSQL becoming more popular.
+
+<!--truncate-->
```

Comment:
Hmm :) With this truncate the lead looks a bit confusing to me, as we don't really tell the audience what we are going to cover or something.

Reply:
For search engines, the "description" in front matter covers what goes into search engine meta description. Truncate only helps to ensure the whole blog post is not displayed on the blog homepage.